src/applications/personalization/profile-2/components/EmailAddressNotification.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import recordEvent from 'platform/monitoring/record-event';
+
+const EmailAddressNotification = ({ signInServiceName }) => {
+  let link;
+  let buttonText;
+
+  if (signInServiceName === 'idme') {
+    link = 'https://wallet.id.me/settings';
+    buttonText = 'ID.me';
+  }
+
+  if (signInServiceName === 'dslogon') {
+    link = 'https://myaccess.dmdc.osd.mil/identitymanagement';
+    buttonText = 'DS Logon';
+  }
+
+  if (signInServiceName === 'mhv') {
```

Comment:
@erikphansen Do you believe these signInServiceNames look right? I was only able to log in with `idme` - took the other variables from `platform/monitoring/external-services/config.js`.

Reply:
I believe so. Just for awareness, we have constants set [here](https://github.com/department-of-veterans-affairs/vets-website/blob/12e4c8326b80fa4fcf3ebeda5de01dd835aae74a/src/platform/monitoring/external-services/config.js) but I think that it's just coincidence that the strings match what you are using here.

Reply:
Thank you! 👍